daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -83,38 +96,198 @@ spec:
     value: "group1"
   - name: clientID # Optional. Used as client tracing ID by Kafka brokers.
     value: "my-dapr-app-id"
-  - name: authRequired # Required.
-    value: "true"
-  - name: saslUsername # Required if authRequired is `true`.
-    value: "adminuser"
+  - name: authType # Required.
+    value: "none"
+  - name: maxMessageBytes # Optional.
+    value: 1024
   - name: consumeRetryInterval # Optional.
     value: 200ms
   - name: version # Optional.
     value: 0.10.2.0
-  - name: saslPassword # Required if authRequired is `true`.
+  - name: disableTls 
+    value: "true"
+```
+
+#### SASL Password
+
+Setting `authType` to `password` will enable [SASL](https://en.wikipedia.org/wiki/Simple_Authentication_and_Security_Layer) authentication using the **PLAIN** mechanism. This requires setting
+the `saslUsername` and `saslPassword` fields. 
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: kafka-pubsub-sasl
+  namespace: default
+spec:
+  type: pubsub.kafka
+  version: v1
+  metadata:
+  - name: brokers # Required. Kafka broker connection setting
+    value: "dapr-kafka.myapp.svc.cluster.local:9092"
+  - name: consumerGroup # Optional. Used for input bindings.
+    value: "group1"
+  - name: clientID # Optional. Used as client tracing ID by Kafka brokers.
+    value: "my-dapr-app-id"
+  - name: authType # Required.
+    value: "password"
+  - name: saslUsername # Required if authType is `password`.
+    value: "adminuser"
+  - name: saslPassword # Required if authType is `password`.
     secretKeyRef:
       name: kafka-secrets
       key: saslPasswordSecret
   - name: maxMessageBytes # Optional.
     value: 1024
-  - name: caCert # Certificate authority certificate.
+  - name: consumeRetryInterval # Optional.
+    value: 200ms
+  - name: version # Optional.
+    value: 0.10.2.0
+  - name: caCert
     secretKeyRef:
       name: kafka-tls
       key: caCert
-  - name: clientCert # Client certificate.
+```
+
+#### Mutual TLS
+
+Setting `authType` to `mtls` will use a x509 client certificate (the `clientCert` field) and key (the `clientKey` field) to authenticate. Note that mTLS as an
+authentication mechanism is distinct from using TLS to secure the transport layer via encryption. mTLS requires TLS transport (meaning `disableTls` must be `false`), but securing
+the transport layer does not require using mTLS. See _Communication using TLS_ for configuring underlying TLS transport.
```

Comment:
Is this "See _Communication using TLS_" supposed to be a link?

Reply:
Yes, I couldn't remember how to do internal links in MD.. Updated now.